lib/jobba.rb

```diff
@@ -1,3 +1,6 @@
+require 'forwardable'
+require 'securerandom'
```

Comment:
I think so, since Rails is not required... specs were failing without those for me.